src/intl/messages.js

```diff
@@ -167,6 +167,7 @@ export const messages: { [messageId: string]: MessageType } = {
   createVmWizardStepTitleNetwork: 'Networking',
   createVmWizardStepTitleReview: 'Review',
   createVmWizardStepTitleStorage: 'Storage',
+  createVmWizardBasicHostNameRules: "Hostname must be at least one character long and may contain only 'a-z', '0-9', '_', '.' or '-' characters.",
```

Comment:
Please make both messages similarly worded since they appear on the same dialog:
"Hostname must be at least one character long and may contain only 'a-z', '0-9', '_', '.' or '-' characters"
"Please enter a valid virtual machine name. Only lower-case and upper-case letters, numbers, and '_','-','.' are allowed."

I think that "Only lower-case and upper-case letters, numbers, and '_','-','.' are allowed" is preferable.

Reply:
@sgratch Done.